package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stytch",
-  "version": "10.0.1",
+  "version": "10.0.2",
```

Comment:
This definitely should've been a MINOR update, btw. It added an entirely new product. I'd actually recommend we put up another PR as 10.1.0 with no other changes just so people know 10.1 added new features